spec/openapi.yaml

```diff
@@ -5457,9 +5455,6 @@ components:
     chronosscheduleresponse:
       title: chronosscheduleresponse
       type: object
-      required:
```

Comment:
@jamie-thomson-sinch I think this should be left as is we don't own Chronos and seperate to reporting. Please revert 5460-5462 unless there is context I am missing here.

Reply:
<img width="744" alt="Screenshot 2023-10-03 at 2 53 28 pm" src="https://github.com/messagemedia/documentation/assets/116612529/550a2b7f-ff80-4104-98cf-f0ec777e107b">
@liam-herbert-messagemedia The only use of this schema was in our reporting section under "Get active reports" as a response so I removed the required field, however since we don't own it if you think its wise to revert it i can.

Reply:
I think it makes sense to remove required fields from responses

Reply:
Yes that makes sense and given used on our responses this is acceptable to touch ✅